subsys/net/ip/dhcpv4.c

```diff
@@ -1163,6 +1163,10 @@ static void dhcpv4_iface_event_handler(struct net_mgmt_event_callback *cb,
 			iface->config.dhcpv4.state = NET_DHCPV4_RENEWING;
 			NET_DBG("enter state=%s", net_dhcpv4_state_name(
 					iface->config.dhcpv4.state));
+			/* Remove any bound address as interface is gone */
+			if (net_if_ipv4_addr_rm(iface, &iface->config.dhcpv4.requested_ip) == false) {
+				NET_DBG("Failed to remove address");
```

Comment:
Could you use here the same printout as the other address removal prints are using, we save couple of bytes of flash that way. So this print should be changed to
```
	NET_DBG("Failed to remove addr from iface");
```

Reply:
done